### DISCUSSION TITLE
Cannot add blog to download

### DISCUSSION THREAD

**Discussion by grateste1** [TumblThreeApp/TumblThree#470] [Q&A (Question & Answer)]
I already downloaded 1 blog but I think I clicked something wrong because I'm trying to download another blog but now it says "Error 1: Access to the path 'Blogs\Index' is denied." How do I resolve this?

**Comment by mtkft** [comment]
Very bizarre bug. For any future users, and for grateste1 if it still plagues you: I have the application as a directory and it is inexplicably sensitive to the version number in the name of said directory (e.g. I installed it as `Downloads/TumblThree-v2.19.0-x64-Application/` and after updating using the automation, I had to change that to read `[...]v2.20.0[...]`).
